jobs/config.json

```diff
@@ -1917,6 +1917,7 @@
   "ci-kubernetes-e2e-gce-federation": {
     "args": [
       "--check-leaked-resources",
+      "--cluster=prow-us-central1-f",
```

Comment:
I am somewhat worried about this. This will break if we change the cluster zones, change the order of the zones or the prefix. It's Ok for now, but something we need to keep in mind.